### PR TITLE
Spread - Ninth and Tenth test both working

### PR DESCRIPTION
**Pull request by fvalles**
filterAttribs filters \'a\' and \'b\' by default

filterAttribs does not mutate the argument

### CODE REVIEW

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 1]
```diff
@@ -1,10 +1,57 @@
...
+    let copiedArray = [...args];
+    return copiedArray;
+  } else {
+    let copiedObj = Object.assign({}, args);
+    return copiedObj;
+  }
+}
 
+export const reverseMerge = (firstArray, secondArray) => {
+  let revMergedArray = [...secondArray, ...firstArray];
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: 👌

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 2]
```diff
@@ -1,10 +1,57 @@
...
+    return args[0];
+  } else {
+    return undefined;
+  }
 }
 
-export function copy() {
+export function copy(args) {
+  if (Array.isArray(args)) {
+    let copiedArray = [...args];
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: 👌

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 3]
```diff
@@ -1,10 +1,57 @@
...
+    return copiedArray;
+  } else {
+    let copiedObj = Object.assign({}, args);
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: This case is wrong. U have to use spread operator

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 4]
```diff
@@ -1,10 +1,57 @@
 import { isArray } from './utils';
 
 export function min() {
-  isArray();
-  return Math.min();
+  let args = [...arguments];
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: U have to receive params on the function name using spread

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 5]
```diff
@@ -1,10 +1,57 @@
...
+    args = args[0];
+  }
+  
+  if (args.length > 1) {
+    return Math.min(...args);
+  } else if (args.length === 1) {
+    return args[0];
+  } else {
+    return undefined;
+  }
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: I recommend you that start using ternary operator.  Can u try that here?

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 6]
```diff
@@ -1,10 +1,57 @@
...
+    return copiedObj;
+  }
+}
 
+export const reverseMerge = (firstArray, secondArray) => {
+  let revMergedArray = [...secondArray, ...firstArray];
+  return revMergedArray;
 }
+
+export const filterAttribs = (obj) => {
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: All this function is wrong. U have to use spread operator to filter attrs

File: js finger exercises/react/ecmascript/01 - spread/index.js (hunk 4 above)
Comment (fvalles) [reply js finger exercises/react/ecmascript/01 - spread/index.js]: Fixed!

File: js finger exercises/react/ecmascript/01 - spread/index.js (hunk 3 above)
Comment (fvalles) [reply js finger exercises/react/ecmascript/01 - spread/index.js]: Fixed!

File: js finger exercises/react/ecmascript/01 - spread/index.js (hunk 5 above)
Comment (fvalles) [reply js finger exercises/react/ecmascript/01 - spread/index.js]: When I changed the code of min() I also modified the conditional part. In the current function there isn't a single "if-else" statement in which I could use the ternary operator :(. I'll take the advice anyway!

File: js finger exercises/react/ecmascript/01 - spread/index.js (hunk 6 above)
Comment (fvalles) [reply js finger exercises/react/ecmascript/01 - spread/index.js]: Function fixed!

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 7]
```diff
@@ -1,57 +1,39 @@
-import { isArray } from './utils';
-
-export function min() {
-  let args = [...arguments];
+/* import { isArray } from './utils'; */
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: Avoid committing comments like this

File: js finger exercises/react/ecmascript/01 - spread/index.js [hunk 8]
```diff
@@ -1,57 +1,39 @@
...
+  const objKeys = Object.keys(filteredObj);
 
-  let filteredObj = {};
-
-  newObjKeys.forEach((key, index) => {
-    assignKeyValue(filteredObj, key, newObjValues[index])
+  objKeys.forEach(key => {
+    if (key === 'a' || key === 'b') {
+      delete filteredObj[key];
+    }
```
Comment (WFolini) [inline js finger exercises/react/ecmascript/01 - spread/index.js]: Wrong :( Here is an example
```js
const a = { one: 'one', two: 'two', three: 'three' };
const { one, ...filtered } = a; // I'm filtering one
console.log(filtered); // { two: 'two', three: 'three' };

**Pull request merged**